pages/linux/lspci.md

```diff
@@ -0,0 +1,23 @@
+# lspci
+
+> List all PCI devices.
+
+- Show a brief list of devices:
+
+`lspci`
+
+- Display additional info:
+
+`lspci -v`
+
+- Display drivers and modules handling each device:
+
+`lspci -k`
+
+- Show a specific device:
+
+`lspci -s {{bus:device.func}}`
```

Comment:
Where can I get this information? Perhaps a concrete example here would work better?

Reply:
Hi, If you run lspci you get lines like this:
**01:0a.0** Ethernet controller: Realtek Semiconductor Co., Ltd. RTL-8100/8101L/8139 PCI Fast Ethernet Adapter (rev 10)

The highlighted portion shows the bus, device and function values. Yeah maybe concrete example would work better.

Reply:
Ahhh ok.